frontend/src/Metamaps/Import.js

```diff
@@ -388,6 +388,36 @@ const Import = {
       }
     )
   },
+  
+  handleTEXT: function(text, opts = {}) {
```

Comment:
ah I get it now, every other function is like handleCSV handleTSV. I think since it's not an acronym i'll make this change on the branch so it's handleText